webpack.dev.config.js

```diff
@@ -1,41 +1,66 @@
-const webpack = require("webpack");
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const ExtractTextWebpackPlugin = require('extract-text-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const app_root = './src';
+const webpack = require('webpack');
 
-function modifyBabelLoader(loader) {
-  if (loader.loader === 'babel-loader') {
-    loader.options = {
-      babelrc: false,
-      presets: [
-        [ "es2015", { modules: false } ],
-        "stage-0",
-        "react",
-      ],
-      plugins: [
-        "react-hot-loader/babel"
-      ],
-    };
-  }
-
-  return loader;
-}
-
-module.exports = require('./webpack.config.js');
-
-module.exports.devServer.hot = true;
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
require it [here](https://github.com/opstalent/React-Redux-Startekit/pull/4/files#diff-361932ce6b04b8880716e7995633922aR3)